release/utils.py

```diff
@@ -55,12 +52,10 @@ def _is_commit_successful(commit_sha):
         Ensures there is at least one status update so that
         commits whose tests haven't started yet are not valid.
         """
-        statuses = github_api.commit_statuses(commit_sha)
-
-        # Determine if all statuses are success
-        passed_tests = all(status['state'] == 'success' for status in statuses)
+        commit_status = github_api.commit_statuses(commit_sha)
 
-        return len(statuses) == NUMBER_OF_TEST_SUITES and passed_tests
+        # Determine if the commit has passed all checks
+        return commit_status.get('state') == 'success'
```

Comment:
Good point. That should be covered in the tests. The status gets set to 'pending' when builds have started.

Here's a screenshot of a PR running right now:

![image](https://cloud.githubusercontent.com/assets/2004678/12122777/298f264e-b3ab-11e5-9c14-76671137dd80.png)